server.js

```diff
@@ -7,9 +7,18 @@ const getFiles = async ()=>{
     return [indexHtml,indexJs]
 }
 
-
 http.createServer(async ({ url }, res) => {
     const [indexHtml, indexJs] = await getFiles()
+    if (url.includes('form')) {
+        const data = url.match(/[a-zA-Z]+|[0-9]{4}-[0-9]{2}-[0-9]{2}/g)
```

Comment:
Right now the date is not saved in the backend. It is created just for the time of request but later on is deleted. When we reload the page all date is gone.